ha/util/mock_health_event_publisher.py

```diff
@@ -0,0 +1,100 @@
+import sys
+import argparse
+
+from cortx.utils.conf_store import Conf
+from ha.util.conf_store import ConftStoreSearch
+
+
+_index = 'cortx'
+cluster_config_file = '/etc/cortx/cluster.conf'
```

Comment:
Don't we have a constant defined in const file for this already? If not, lets add one and use it here.